.pax/prepare-workspace.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash -e
+
+################################################################################
+# This program and the accompanying materials are made available under the terms of the
+# Eclipse Public License v2.0 which accompanies this distribution, and is available at
+# https://www.eclipse.org/legal/epl-v20.html
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Copyright IBM Corporation 2018, 2020
+################################################################################
+
+################################################################################
+# Build script
+# 
+# - build client
+# - import ui server dependency
+################################################################################
+
+# contants
+SCRIPT_NAME=$(basename "$0")
+BASEDIR=$(dirname "$0")
+PAX_WORKSPACE_DIR=.pax
+PACKAGE_NAME=$(node -e "console.log(require('./package.json').name)")
+PACKAGE_VERSION=$(node -e "console.log(require('./package.json').version)")
+PACKAGE_DESC=$(node -e "console.log(require('./package.json').description)")
+# APPLICAIION_URI=$(node -e "console.log(require('./package.json').config.baseuri)")
+# APPLICAIION_PORT=$(node -e "console.log(require('./package.json').config.port)")
+ZOWE_PLUGIN_ID="com.ibm.${PACKAGE_NAME}"
+
+cd $BASEDIR
+cd ..
+ROOT_DIR=$(pwd)
+
+# prepare pax workspace
+echo "[${SCRIPT_NAME}] cleaning PAX workspace ..."
+rm -fr "${PAX_WORKSPACE_DIR}/content"
+mkdir -p "${PAX_WORKSPACE_DIR}/content"
+
+# build client
+echo "[${SCRIPT_NAME}] building client ..."
+cd "dist"
+npm install
+
+cd "${ROOT_DIR}"
+
+# copy sample-node-api to target folder
+echo "[${SCRIPT_NAME}] copying sample node api backend ..."
+mkdir -p "${PAX_WORKSPACE_DIR}/content"
+cp README.md "${PAX_WORKSPACE_DIR}/content"
+cp package.json "${PAX_WORKSPACE_DIR}/content"
+cp package-lock.json "${PAX_WORKSPACE_DIR}/content"
+cp -r dist/. "${PAX_WORKSPACE_DIR}/content"
```

Comment:
I'm thinking if it's good to also source code, not just `dist` code into the pax.

Normally we don't do that, but this sample api, we suggest the user to customize it on there server, so it's good to have original source code.

Reply:
package json and lock already included, just added readme via build script too